docs/copy-assets.md

```diff
@@ -21,3 +21,55 @@ You can copy these assets by using the `assets` option.
   }
 }
 ```
+
+## Exporting Styles
+If your copied assets include styles that should be part of your library's public API, they need to be represented in the [`exports`](https://angular.io/guide/angular-package-format#exports) of its `package.json`.
+
+To do this, you can enumerate all such exports in an `"exports"` object in your base `package.json`.  `ng-packagr` will augment this and merge the additional exports generated from the build.
+
+### Example `package.json`
+```json
+{
+  "name": "your-library",
+  "version": "1.2.3",
+  "exports": {
+    ".": {
+      "sass": "./_index.scss"
+    },
+    "./styles/dark-theme": {
+      "sass": "./styles/_dark-theme.scss"
+    },
+    "./styles/_dark-theme": {
```

Comment:
Can we remove all the exports with `_`? These are due to maintain legacy backwards compatibility in Material but shouldn’t be needed otherwise.

Reply:
Looks like you missed this comment.

Reply:
Ah, yep.  Sorry, didn't see that one for some reason.  I'll take care of that now.

Reply:
All set